content/influxdb3/cloud-dedicated/admin/account/_index.md

```diff
@@ -33,11 +33,28 @@ Your {{% product-name %}} account is a collection of {{% product-name omit="Clus
 
 ## View account information
 
-After you log in to the Admin UI, the Account Management portal displays the following information about your account:
+The Admin UI provides access to account information through the **All Accounts** page:
 
-- Account ID
-- Contract status
-- Contract start date
-- The [list of clusters](/influxdb3/cloud-dedicated/admin/clusters/list/?t=admin-ui) associated with the account
+1. Click the **Account** selector at the bottom of the sidebar navigation
+2. Select **All Accounts** from the menu
+3. View a table with all accounts across your organization
 
-{{< img-hd src="/img/influxdb3/cloud-dedicated-admin-ui-account-info.png" alt="InfluxDB Cloud Dedicated Admin UI account information" />}}
+{{< img-hd src="/img/influxdb3/cloud-dedicated-admin-ui-all-accounts.png" alt="InfluxDB Cloud Dedicated Admin UI all accounts view" />}}
```

Comment:
For these screenshots, are we omitting the navbar? That's fine with me -- I just wanted to make sure the update took, as I didn't seen the navbar in the screenshot.